portfolio/src/main/webapp/scripts/github/client.js

```diff
@@ -0,0 +1,16 @@
+const baseURL = "https://api.github.com";
+
+const github = {
+  get: async (path) => {
+    const data = await fetch(`${baseURL}${path}`, {
+      headers: {
+        Accept: "application/vnd.github.v3+json",
+      },
+    });
+
+    return data.json();
+  },
+  // TO DO: Add another methods to github
```

Comment:
nit: Might be a bit redundant here, because you're the only one contributing to the project, but it could be good practice to do these in the format we normally do TODOs. Example:

TODO(livvielin): Add foo and bar methods

Usually in the parentheses we use our username, or preferable a bug/issue number to track, and try to be specific in the comments.

Same for other files

Reply:
Perfect!

Changed in a new commit. Also, I have a question. Do you think it's okay to add this TO-DOs? I just added them as an idea, but due to portfolio's nature, seems like I won't ever need a POST method or something.

Reply:
I think it's fine to leave them in, in case you want to come back to it and add it later as an extra thing.